core/server/data/migrations/versions/4.0/06-populate-members-subscribe-events-table.js

```diff
@@ -0,0 +1,41 @@
+const ObjectID = require('bson-objectid');
+const {createTransactionalMigration} = require('../../utils');
+
+module.exports = createTransactionalMigration(
+    async function up(knex) {
+        const allMembers = await knex.select(
+            'id as member_id',
+            'created_at'
+        ).from('members');
+
+        const unsubscribedMembers = await knex.select(
+            'id as member_id',
+            'updated_at as created_at'
```

Comment:
Makes the assumption that the last time the member was updated was when they unsubscribed - not accurate, but it'll do IMO